package.json

```diff
@@ -35,7 +60,7 @@
     "build:browser:esm": "FMT=esm ENV=browser rollup -c --name ulidx",
     "build:node:cjs": "FMT=cjs ENV=node rollup -c",
     "build:node:esm": "FMT=esm ENV=node rollup -c",
-    "build:types": "tsc -p tsconfig.dec.json --emitDeclarationOnly",
+    "build:types": "tsc -p tsconfig.dec.json --emitDeclarationOnly && find ./dist -name '*.d.ts' -exec sh -c 'cp {} $(dirname {})/$(basename -s .d.ts {}).d.cts' \\;",
```

Comment:
This doesn't make sense to me, as the `.cts` type is superfluous here. We could simply point `require.types` under any export to the `index.d.ts` file as the contents are the same. If the CTS extension were valuable here, wouldn't we be using common-js style exports in the `.d.ts` file? And I'm not even sure that such a thing has ever existed.

Reply:
Nevermind, I read up on the `cts` extension and stand corrected.